include/avif/internal.h

```diff
@@ -15,26 +15,44 @@ extern "C" {
 #define AVIF_MIN(a, b) (((a) < (b)) ? (a) : (b))
 #define AVIF_MAX(a, b) (((a) > (b)) ? (a) : (b))
 
+// Used for debugging. Add ! to catch the earliest failure during encoding or decoding.
+#if defined(AVIF_BREAK_ON_ERROR)
+static inline void avifBreakOnError()
+{
+    // Same mechanism as OpenCV's error() function.
+    static volatile int * p = 0;
```

Comment:
`p` (a pointer) should be initialized to `NULL`.

The `static` and `volatile` keywords should be unnecessary. I often use the same code (without those two keywords) to force a crash.